trucks.js

```diff
@@ -246,7 +249,40 @@ var foodTrucks = [
   }
 ];
 
-/* 
+
+var trucks = (function(){
+
+  var truckString = '';
+  var trucks = [];
+
+  function getTruckByDay(truck,day){
+    if(truck.schedule.indexOf(day) !== -1)
+      return true;
+    else
+      return false;
+  }
+  function filterTrucksByDay(day){
```

Comment:
This function would be better just returning an array of truck objects. You don't want to assume how the consumer of the function will use the information.